content/docs/api-reference/events/trigger/delete.mdx

```diff
@@ -21,4 +16,9 @@ _openapi:
 
 {/* This file was generated by Fumadocs. Do not edit this file directly. Any changes should be made by running the generation command again. */}
 
+
+    Using a previously generated transactionId during the event trigger,
+     will cancel any active or pending workflows. This is useful to cancel active digests, delays etc...
```

Comment:
Can we add the info "If workflow execution is complete, there will not be any effect"

Reply:
Defient;y, but will need to do that on the API sevice